dev/doxygen/pages/building.md

```diff
@@ -0,0 +1,99 @@
+Building LibrePCB {#doc_building}
+=================================
+
+[TOC]
+
+This page is aimed both at developers as well as packagers. If you're a
+packager and have any question regarding the LibrePCB build system, please join
+our [chat or forum](https://librepcb.org/discuss/)!
+
+
+# Building with Qt Creator {#doc_building_qtcreator}
+
+If you want to contribute a new feature or a bugfix to LibrePCB, the best way
+is to use [Qt Creator](https://doc.qt.io/qtcreator/). To get started, simply
+open the `librepcb.pro` file with Qt Creator.
+
+## Configuring the Desktop Kit
+
+When opening a project in Qt Creator for the first time, you need to configure
+the Desktop kit:
+
+![qtcreator_desktopkit](qtcreator_desktopkit.png)
+
+Click on the "Configure Project" button to get started.
+
+## Starting LibrePCB
+
+Select the run configuration `librepcb` and click on the `Run` button:
+
+![qtcreator_run](qtcreator_run.png)
+
+## Parallel Builds
+
+To keep build time as low as possible make sure to set the correct make flags
+to use all available CPU cores to build. See this [stackoverflow
+answer](https://stackoverflow.com/questions/8860712/setting-default-make-options-for-qt-creator).
+
+
+# Building on the Command Line {#doc_building_command_line}
+
+You can also build LibrePCB from the command line using qmake:
+
+    mkdir build && cd build
+    qmake -r ../librepcb.pro
+    make -j8
+
+Then run the `librepcb` library in the `build/output/` directory.
```

Comment:
```suggestion
Then run the `librepcb` application in the `build/output/` directory.
```

Reply:
Changed to `binary`, which is what I intended to write...